code/game/gamemodes/objective.dm

```diff
@@ -876,7 +885,34 @@ GLOBAL_LIST_EMPTY(possible_items_special)
 					stolen_count++ //it counts. nice.
 			else if(is_type_in_typecache(I, wanted_items))
 				stolen_count++
-	return stolen_count >= 5
+	return stolen_count >= amount
+
+/datum/objective/steal_n_of_type/organs
+	name = "steal organs"
+	explanation_text = "Steal at least 10 organs! They must be kept healthy."
+	wanted_items = list(/obj/item/organ)
```

Comment:
If this includes cyberorgans, stealing these will just be absurdly easy (and won't have decay). You can print them at roundstart, even.

Reply:
agreed, will have to ban that